ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -26,6 +28,12 @@ export default function FeeCard({
     bestQuoteText = t('swapBetterQuoteAvailable');
   }
 
+  const getTranslatedNetworkName = () => {
+    const networkNameKey =
+      SWAPS_CHAIN_ID_TO_NETWORK_NAME_KEY_MAP[chainId] || 'networkNameEthereum';
+    return t(networkNameKey) || 'Ethereum'; // If there are no translated network names, it defaults to 'Ethereum'
```

Comment:
The translation system already defaults to using the `en` translation - this fallback is redundant.

Reply:
Awesome, will remove it.

Reply:
Done.